### PR TITLE
Remove FullIO from utilities

### PR DESCRIPTION
**Pull request by OznOg**
This class is almost not used and is not portable.

This patch reword the only test that use it and remove the fullIO code.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/203%23issuecomment-139222112%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/203%23discussion_r39154187%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/203%23issuecomment-139222353%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/203%23issuecomment-139222112%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-10T12%3A35%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-10T12%3A36%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2098964bd39c9e45df0f3e59ced8ae96d8499c27c8%20remote-processor/CMakeLists.txt%2018%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/203%23discussion_r39154187%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Maybe%20add%20your%20comment%20as%20error%20message%20if%20ASIO_DIR%20is%20not%20found%20%3F%22%2C%20%22created_at%22%3A%20%222015-09-10T12%3A36%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20remote-processor/CMakeLists.txt%3AL35-52%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 98964bd39c9e45df0f3e59ced8ae96d8499c27c8 remote-processor/CMakeLists.txt 18'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/203#discussion_r39154187'>File: remote-processor/CMakeLists.txt:L35-52</a></b>
- <a href='https://github.com/krocard'></a> Maybe add your comment as error message if ASIO_DIR is not found ?

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/203?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/203?approve=1'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/203'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/203'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by krocard** [comment]
:+1:

File: remote-processor/CMakeLists.txt [hunk 1]
```diff
@@ -38,8 +35,17 @@ add_library(remote-processor SHARED
 set(CMAKE_THREAD_PREFER_PTHREAD 1)
 find_package(Threads REQUIRED)
 
+# Find ASIO (the standalone version, not Boost) and declare it as a dependency
+# for remote-processor's users too. (FIXME: this should not be the case)
+# If asio isn't installed in a standard directory, add the correct directory to
+# CMAKE_PREFIX_PATH (see the main README for more information).
+find_path(ASIO_DIR NAMES asio.hpp)
```
Comment (krocard) [inline remote-processor/CMakeLists.txt]: Maybe add your comment as error message if ASIO_DIR is not found ?

**Comment by krocard** [comment]
:+1:

**Pull request merged**